docs/basics.md

```diff
@@ -312,7 +376,7 @@ The reference to validator and exact trust root is resolved in the following way
 Let's review the pattern and validator matching at a minimal example.
 We consider the following validator and policy configuration (most fields have been omitted for clarity):
 
-```yaml
+```yaml title="helm/values.yaml"
```

Comment:
Not sure how this is rendered, but to keep things consistent, wouldn't it be better to use the same syntax above and below with the

> ??? abstract "helm/values.yaml"
    ```yaml

boxes?

Reply:
`mkdocs serve` to see how `!!! `, `???` and `title="helm/values.yaml"` are rendered. I was uncertain as well and decided to go by "is the output part of the point here or just FYI. Feel free to share your thoughts for a general strategy

Reply:
I think it's inconsistent. Would vote for adding title="helm/values.yaml" everywhere and making the box title something like "Example configuration" or similar

Reply:
then it becomes "title example, box example, ...". If inconsistency is the fear, will duplicate